fpsim/scenarios.py

```diff
@@ -142,49 +114,21 @@ def __init__(self, spec=None, label=None, pars=None, year=None, matrix=None, age
         self.specs = sc.mergelists(*[Scenario(**s).specs for s in sc.tolist(spec)]) # Sorry
         self.label = label
         self.pars  = sc.mergedicts(pars)
-        if not isinstance(label, (str, type(None))):
-            errormsg = f'Unexpected label type {type(label)}'
-            raise TypeError(errormsg)
 
         # Handle other keyword inputs
-        eff_spec   = None
-        prob_spec  = None
+        contraceptive_spec = None
         par_spec   = None
         intv_specs = None
 
```

Comment:
Seems like this particular script has lost quite a bit of documentation. Noting for myself to return to later.